csharp/ql/test/library-tests/expressions/ArrayCreation11.expected

```diff
@@ -0,0 +1,3 @@
+| expressions.cs:168:27:168:44 | array creation of type Object[] | expressions.cs:168:27:168:44 | 1 | true |
+| expressions.cs:409:23:409:65 | array creation of type Int32[,] | expressions.cs:409:23:409:65 | 3 | true |
+| expressions.cs:409:23:409:65 | array creation of type Int32[,] | expressions.cs:409:23:409:65 | 3 | true |
```

Comment:
This shows a bug. I'll create a ticket for it. The array is actually `new int[3,2]` and not `new int[3,3]`.